packages/mongo-nav/src/org-nav/GovBanner.tsx

```diff
@@ -0,0 +1,149 @@
+import React from 'react';
+import GovBuildingIcon from '@leafygreen-ui/icon/dist/GovBuilding';
+import Tooltip from '@leafygreen-ui/tooltip';
+import { css, cx } from '@leafygreen-ui/emotion';
+import { uiColors } from '@leafygreen-ui/palette';
+import { Body } from '@leafygreen-ui/typography';
+
+const desktopContainer = css`
+  margin-bottom: 10px;
+  position: relative;
+  z-index: 2;
+  display: flex;
+  flex-grow: 1;
+  height: 5px;
+  background-color: #edf5fa;
+  align-items: flex-start;
+  justify-content: center;
+`;
+
+const tabletContainer = css`
+  position: relative;
+  z-index: 2;
+  display: flex;
+  flex-grow: 1;
+  align-items: flex-start;
+  justify-content: center;
+  height: 0;
+`;
+
+const mobileContainer = css`
+  width: 30px;
+  height: 30px;
+  background-color: #edf5fa;
+  margin-left: 12px;
+  border-radius: 7px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  cursor: pointer;
+`;
+
+const sharedBannerContainerStyles = css`
+  background-color: #edf5fa;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  cursor: pointer;
+  position: relative;
+`;
+
+const desktopTooltipContainer = css`
+  height: 34px;
+  width: 205px;
+  border-radius: 0 0 17px 17px;
+`;
+
+const tabletTooltipContainer = css`
+  height: 20px;
+  width: 200px;
+  border-radius: 0 0 15px 15px;
+`;
+
+const tooltipWidth = css`
+  width: 315px;
+`;
+
+const iconColor = css`
+  color: ${uiColors.blue.base};
+`;
+
+const bannerMargin = css`
+  margin-bottom: 4px;
+  margin-right: 2px;
+`;
+
+const mobileMargin = css`
+  margin-bottom: 4px;
+  margin-top: 4px;
+`;
+
+const bodyStyle = css`
+  font-size: 11px;
+  color: #0d324f;
+`;
+
+const tooltipContent = (
+  <div className={tooltipWidth}>
+    You are currently in the MongoDB Cloud for Government environment that is
+    FedRAMP authorized.
+  </div>
+);
+
+const sharedTooltipProps = {
+  align: 'bottom',
```

Comment:
`justify: 'middle'`?

Reply:
`justify: 'end'` for mobile

Reply:
Do we want to specify this here?

Reply:
Do you mean that I should add a comment specifying that the justify position is defined separately on the banner and tooltip bc its not a common value across both components?